src/renderer/services/plugin-manager.js

```diff
@@ -743,13 +743,19 @@ class PluginManager {
         },
 
         setInterval (...args) {
-          const id = setInterval(...args)
+          const functionArgs = args.splice(2)
+          const id = setInterval(function () {
+            args[0](...functionArgs)
+          }, args[1])
           timerArrays.intervals.push(id)
           return id
         },
 
         setTimeout (...args) {
```

Comment:
Same here (e.g. `callback` and `timeout`)

Reply:
Done, using `method` instead of `callback` to prevent the `no-callback-literal` error in eslint.